### PR TITLE
dev-python/pipenv: fix outdated dependency version

### PR DESCRIPTION
**Pull request by oz123**
Bug: https://bugs.gentoo.org/645530
Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

File: dev-python/pipenv/pipenv-9.0.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.python.org/pypi/pipenv"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
```
Comment (titanofold) [inline dev-python/pipenv/pipenv-9.0.0-r1.ebuild]: Delete empty vars.

File: dev-python/pipenv/pipenv-9.0.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	>=dev-python/flake8-3.0.0[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-python/pew-0.1.26[${PYTHON_USEDEP}]
+	>=dev-python/pip-9.0.1[${PYTHON_USEDEP}]
+	>dev-python/requests-2.18.0[${PYTHON_USEDEP}]
```
Comment (titanofold) [inline dev-python/pipenv/pipenv-9.0.0-r1.ebuild]: Typo? Should it be `>=`?

Comment (oz123) [reply dev-python/pipenv/pipenv-9.0.0-r1.ebuild]: @titanofold , I copied this from the previous ebuild, my intent was only to address the bug mentioned, not to fix other QA issues.
However, this seems correct to me. That is at list [how upstream defines this dependency][1].

[1]: https://github.com/pypa/pipenv/blob/e8e4899a1df5f8f5cd8f4ee7f7f4574a0efacbb9/setup.py#L28

**Comment by oz123** [comment]
@titanofold ping. Is there something else I can do to merge this PR?

**Comment by mgorny** [comment]
@oz123, to avoid this in the future, please paste `diff -dup` between the old ebuild and the new one. GitHub makes it hard to review changes which involve a revision bump.

**Comment by mgorny** [comment]
(in a comment, that is)

**Comment by oz123** [comment]
@mgorny , yes, you are right about this. I was wondering how this can be improved.

```
$ diff -dup pipenv-9.0.0.ebuild pipenv-9.0.0-r1.ebuild 
--- pipenv-9.0.0.ebuild	2017-12-12 12:19:37.364009252 +0100
+++ pipenv-9.0.0-r1.ebuild	2018-01-25 22:49:01.156000929 +0100
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
@@ -16,9 +16,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="test"
 
-RDEPEND=""
-DEPEND="${RDEPEND}
-	dev-python/flake8
+DEPEND="
+	>=dev-python/flake8-3.0.0[${PYTHON_USEDEP}]
 	dev-python/setuptools[${PYTHON_USEDEP}]
 	>=dev-python/pew-0.1.26[${PYTHON_USEDEP}]
 	>=dev-python/pip-9.0.1[${PYTHON_USEDEP}]
```

**Pull request closed without merging**